### PR TITLE
Use Maven wrapper

### PR DESCRIPTION
**Pull request by bjorncs**
I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

**Comment by bjorncs** [comment]
FYI @gjoranv

File: docker/include/populate-m2-repo.sh [hunk 1]
```diff
@@ -21,9 +21,12 @@ if [[ -n $(find /opt/rh -mindepth 1 -maxdepth 1 -type d -name "rh-maven*") ]]; t
   source /opt/rh/rh-maven*/enable
 fi
 
+readonly MVNW=/opt/vespa-systemtests/tests/mvnw
```
Comment (aressem) [inline docker/include/populate-m2-repo.sh:24]: How do you know that this location is writable by the user ? And I dont see how we end up in this dir. I propose to use a temp location and remove the wrapper afterwards.

**Review by aressem — CHANGES_REQUESTED** [review]

**Review by aressem — APPROVED** [review]

File: docker/include/populate-m2-repo.sh [hunk 2]
```diff
@@ -21,12 +21,16 @@ if [[ -n $(find /opt/rh -mindepth 1 -maxdepth 1 -type d -name "rh-maven*") ]]; t
...
 
-readonly MVNW=/opt/vespa-systemtests/tests/mvnw
+readonly MVNW=/tmp/mvnw
 readonly MVN_VERSION=3.8.8
-readonly SHARED_MVN_OPTS="--threads 1C -Dvespa.version=${VESPA_VERSION} -Dmaven.repo.local=${LOCAL_M2_REPO} --batch-mode --file /opt/vespa-systemtests/tests/pom.xml"
+readonly TESTS_ROOT=/opt/vespa-systemtests
+readonly SHARED_MVN_OPTS="--threads 1C -Dvespa.version=${VESPA_VERSION} -Dmaven.repo.local=${LOCAL_M2_REPO} --batch-mode --file ${TESTS_ROOT}/tests/pom.xml"
 # Install Maven Wrapper
-mvn $SHARED_MVN_OPTS --show-version wrapper:wrapper -Dmaven=$MVN_VERSION
+cp $TESTS_ROOT/tests/pom.xml /tmp/pom.parent.xml
```
Comment (gjoranv) [inline docker/include/populate-m2-repo.sh:29]: I don't think you have to copy the pom file. Try to just read it from where it is.

Comment (bjorncs) [reply docker/include/populate-m2-repo.sh:29]: We don't want to mvnw binary to be created inside the source code directory.

**Pull request merged**